### DISCUSSION TITLE
geopandas Cheat Sheet

### DISCUSSION THREAD

**Discussion by atseewal** [geopandas/geopandas#2772] [Show and tell]
Hi everyone,

I created a geopandas cheat sheet. It's heavily based on the [sf cheat sheet](https://github.com/rstudio/cheatsheets/blob/main/sf.pdf) for spatial manipulation in R. I used the [template](https://github.com/rstudio/cheatsheets/blob/main/.github/CONTRIBUTING.md#contribute-a-new-cheat-sheet) from Posit (RStudio) to make it.

I was hoping for some feedback on the cheat sheet from the community. I find them super helpful for other packages and would love to hear what everyone else thinks!

[Link to the cheat sheet PDF on my GitHub](https://github.com/atseewal/cheatsheets/blob/main/Geopandas.pdf)

![image](https://user-images.githubusercontent.com/39170343/215913254-8dbaec57-8ddd-4964-98e5-db30ff65eedf.png)
![image](https://user-images.githubusercontent.com/39170343/215913305-e4e317bc-ba70-4664-905d-9a29772c48e1.png)

**Comment by camwest5** [comment]
Hi @atseewal, it looks like the link to the PDF is broken.

**Reply by martinfleis** [reply]
Looks it was just renamed - https://github.com/atseewal/cheatsheets/blob/main/Geopandas.pdf

**Comment by ichibanbrisket** [comment]
Thnx I needed this
